versions.properties

```diff
@@ -1,5 +1,5 @@
 #
-# Copyright � 2022 Lunabee Studio
+# Copyright � 2022 Lunabee Studio
```

Comment:
again 🤔

Reply:
I think it is changed by refreshVersion each time 😬